### DISCUSSION TITLE
[Question][Hw5] Questions regarding binary operations on non-numerical values

### DISCUSSION THREAD

**Discussion by f52985** [prosyslab-classroom/cs524-program-analysis#329] [Q&A]
I've checked that 

> The other comparison operators, such as <, are not considered for non-numerical values", 

and there are some questions regarding it.

1. Do we have to consider the case for comparison operators == and !=, where one side of the comparison is numerical and the other side of the comparison is non-numerical? If so, what should be the result of the comparison? (i.e. What will the result of `0 == NULL` ?)
2. Do we have to consider the case for comparison operators == and !=, where one side of the comparison is a location and the other side of the comparison is function? If so, what should be the result of the comparison? 
3. Do we have to consider arithmetic binary operators for non-numerical values? If so, what should be the result?

**Comment by KAIST-JongchanPark** [comment]
1, 2 )  
Please ignore those cases.

You can consider only for comparison operators between same types.

3 ) You can ignore those cases.
